pkg/controllers/disruption/controller.go

```diff
@@ -152,6 +152,11 @@ func (c *Controller) disrupt(ctx context.Context, disruption Method) (bool, erro
 	if err != nil {
 		return false, fmt.Errorf("determining candidates, %w", err)
 	}
+	EligibleNodesGauge.With(map[string]string{
```

Comment:
Do we have testing in place for this metric to ensure we don't regress on this issue in the future?

Reply:
Not at the moment, but I'll go ahead and add some 👍